iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -65,7 +66,7 @@ void publisher()
     {
         publisher.loan().and_then([&](auto& sample) {
             sample->counter = counter++;
-            consoleOutput(std::string("Sending   " + greenRightArrow + std::to_string(sample->counter)));
+            consoleOutput(std::string("Sending   " + greenRightArrow + iox::cxx::convert::toString(sample->counter)));
```

Comment:
We could get completely rid of `std::string` by adjusting the `consoleOutput` funtion a little bit
```cpp
void consoleOutput(const char* source, const char* arrow, const uint64_t counter) {...}
```
The `greenRightArrow` would then also become a `constexpr const char[] = "..."`

Reply:
yeah.. will do this